sphinxext/opengraph/descriptionparser.py

```diff
@@ -124,5 +124,4 @@ def get_description(
 
     mcv = DescriptionParser(description_length, known_titles, document)
     doctree.walkabout(mcv)
-    # Parse quotation so they won't break html tags if smart quotes are disabled
-    return mcv.description.replace('"', "&quot;")
+    return mcv.description
```

Comment:
Why was this changed?

Reply:
This wasn't removed but moved to __init__.py (make_tag function) so it would get applied on all the created tags, including the ones added via field lists.